runtime-light/stdlib/serialization/msgpack-functions.h

```diff
@@ -0,0 +1,60 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2025 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include "runtime-common/stdlib/serialization/msgpack-functions.h"
+
+template<class ResultType = mixed>
+ResultType f$msgpack_deserialize(const string &buffer, string *out_err_msg = nullptr) noexcept {
```

Comment:
Is there a reason for this function to not be implemented in `runtime-common`?

Reply:
Yeah, in old runtime it use `make_malloc_replacement_with_script_allocator`, which doesn't exists in runtime-light